jdav_client/yew_styles.css

```diff
@@ -3,7 +3,11 @@
 }
 
 .scrolllist {
+  height: 60vh;
   overflow-y: scroll;
+}
+
+.overview {
```

Comment:
Move this class to another css file and revert changes to this file which is provided by `yew-styles` and will be updated with every update of the library.